.circleci/config.yml

```diff
@@ -64,7 +64,9 @@ jobs:
       - run:
           name: Dry run queries
           command: |
-            if [ "$CIRCLE_BRANCH" = master ]; then
+            if [ "$CIRCLE_BRANCH" = master ] ||
+                git log --format=%B --no-merges -n 1 |
+                grep -qF '[run-tests]'; then
```

Comment:
What's going on here? It's parsing the tag to run all the tests regardless of branch?